specs/manage_tokens_list/01_main.rst

```diff
@@ -0,0 +1,137 @@
+==========================================================
+Manage Tokens - Selection List
+==========================================================
+
+=======  ====================  =======    ==========
+ epic          title           author     created
+=======  ====================  =======    ==========
+58_      Token Selection List  rmeissner  23.01.2019
+=======  ====================  =======    ==========
+
+.. _58: https://github.com/gnosis/safe/issues/58
+
+.. _Main:
+
+#. `Main`_
+#. `User Interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+-------------------------------
+
+- User MUST be able to see all token supported by the Safe app
+- User MUST be able to enable and disable tokens that should be displayed in the asset overview 
+
+
+Outputs
+------------
+
+- Selected tokens for asset overview
+
+Use Case Scenarios
+-----------------------
+
+Load token list
+~~~~~~~~~~~~~~~~~~~~
+
+Steps
++++++
+1. The content (token list) should start loading automatically
+2. Once loading is done the tokens should be displayed
+
+Postconditions
+++++++++++++++
+- All available tokens should be displayed after the selected tokens.
+
+  * It should be possible to display an unlimited number of tokens in the list (paginated list)
```

Comment:
IMO this is a UX decision as well. This approach has inconveniences. How the tokens should be sorted? How much should you scroll to a needed token?
I would prefer to not have pagination. 100-300 most used tokens and a search bar can work as well.